osquery/tables/system/linux/system_info.cpp

```diff
@@ -57,19 +59,24 @@ QueryData genSystemInfo(QueryContext& context) {
     r["physical_memory"] = "-1";
   }
 
-  r["cpu_type"] = "0";
   r["cpu_subtype"] = "0";
 
+  struct utsname utsbuf;
+  if (uname(&utsbuf) == -1) {
+    VLOG(1) << "Failed to get cpu_type, uname failed with error code: " << std::to_string(errno);
```

Comment:
How often might this happen? I feel like this might be better as a `LOG(WARNING)`, as it's a failure to retrieve data.